qiskit_optimization/translators/ising.py

```diff
@@ -184,9 +184,9 @@ def from_ising(
         warn(
             "The `qubit_op` argument can currently accept Opflow operators (`OperatorBase` type), "
             "but have been superseded by Qiskit Terra quantum_info `BaseOperators` such as "
-            "`SparsePauliOp`. Opflow operator support will be deprecated in a future release "
+            "`SparsePauliOp`. Opflow operator has been deprecated in Qiskit Terra 0.25.0 "
```

Comment:
```suggestion
            "`SparsePauliOp`. Opflow operator has been deprecated in Qiskit Terra 0.24.0 "
```
Opflow is deprecated in the imminent release of Terra which is 0.24.0

Reply:
Hmm... I see why you put 0.25 as that is what all the deprecation messages - I need to look into that! We do build off main and the version has been already bumped there since the prep and release of 0.24rc1 last week and the release of 0.24 shortly.